Formula/openssl.rb

```diff
@@ -67,58 +63,18 @@ def install
     dirs = []
 
     archs.each do |arch|
```

Comment:
Technically there is still two here, because he's left in the possibility the CPU being built against is 32-bit, just above this blob.